yield_plugin/src/yield_plugin.cpp

```diff
@@ -258,26 +263,35 @@ namespace yield_plugin
     carma_planning_msgs::msg::TrajectoryPlan original_trajectory = req->initial_trajectory_plan;
     carma_planning_msgs::msg::TrajectoryPlan yield_trajectory;
 
+
+    double initial_velocity = req->vehicle_state.longitudinal_vel;
+    // If vehicle_state is stopped, non-zero velocity from the trajectory
+    // should be used. Otherwise, vehicle will not move.
+    if (initial_velocity < EPSILON)
+    {
+      initial_velocity = original_trajectory.initial_longitudinal_velocity;
```

Comment:
This is a small unrelated bug I found if obstacle is placed near the starting point of the vehicle, the vehicle would not move although it has plenty runway until stopping before the obstacle. This is because `vehicle_state.longitudinal_vel` is 0 when stopped and trajectory generated from yield starts from that speed. Inlanecruising_plugin also had similar problem, which we resolved back in the day, by forcing the first commanded speed to be non-zero, which is where the `initial_longitudinal_velocity` partially came from.